features/org.obeonetwork.m2doc.feature/feature.xml

```diff
@@ -245,21 +245,28 @@ litigation.
          unpack="false"/>
 
    <plugin
-         id="org.obeonetwork.m2doc.ui.edit"

```

Comment:
I think we should keep the ui in a separate feature. In case we integrate M2Doc in a workbench, we might want to create UI actions that are specific to the workbench and hide the generic actions. 

But let's keep this as is for now.